# Add Message header
Header header

# Defines the velocity scale for safety requirement.
float64 scale
